gradle.properties

```diff
@@ -1,3 +1,4 @@
 slf4jVersion=1.7.12
 groovyVersion=2.4.7
```

Comment:
If you're using Spock `2.0-groovy-3.0` doesn't this need to be Groovy 3?

Reply:
That wasn't a used property anymore removed.